src/app.ts

```diff
@@ -13,14 +13,37 @@ interface TemperatureSummary {
   average: number
 }
 
+let readingsGlobal: TemperatureReading[]
 export function processReadings(readings: TemperatureReading[]): void {
   // add here your code
-}
 
+  readingsGlobal = readings
+  return
+}
 export function getTemperatureSummary(
   date: Date,
   city: string,
 ): TemperatureSummary | null {
   //add here your code
-  return null
+  const filter = readingsGlobal.filter(
```

Comment:
use a more descriptive name like ` filteredReadings, filteredList....`

Reply:
I forgot to rename it, I'll do it right away